sys-apps/polkit/instructions.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+after_install() {
+    groupadd -fg 27 polkitd
```

Comment:
```suggestion
    groupadd -f polkitd
```
I don't know why the LFS book does this, but forcing the UID/GID group with groupadd is usually much meh :/

Reply:
sometimes even the Bible is wrong